core/templates/pages/contributor-dashboard-page/contributor-dashboard-page.component.ts

```diff
@@ -112,7 +112,15 @@ export class ContributorDashboardPageComponent
     const activeSuggestionType =
       this.contributionAndReviewService.getActiveSuggestionType();
     const activeTabType = this.contributionAndReviewService.getActiveTabType();
+
+    const isReviewingQuestionSuggestions = (
```

Comment:
Typically we try not to start booleans with a verb, to distinguish them from functions. Can you rename this to userIsReviewingQuestionSuggestions?

Also, would it be better to have a similar clause for translations so that the return statement below has parity between the two?

Reply:
Hmm. It's strange, I would expect boolean value with a prefix `is` to be perfectly normal. But I don't mind renaming. Added a similar variable for the translation part, thanks!

Reply:
Thanks! Yup, it's often ambiguous. Just so you know -- in this codebase we use isXxx() or areXxx() for functions/methods only, and avoid doing that for boolean variables (often changing to xxxIsYyy instead).